core/src/edu/asu/c3simulator/screens/CouncilPanel.java

```diff
@@ -0,0 +1,46 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+/**
+ * Navigation Panel for council screens
+ * 
+ * @author Alyahya, Mohammed
```

Comment:
I can verify that. We created it in Justin's share branch when we were pair programming on linking the screens